tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+    "compilerOptions": {
+        "experimentalDecorators": true,
```

Comment:
@TITAN9389 Do we use TS somewhere? why TSConfig?

Reply:
its for VSCode just - always throws error on Decorators.